lib/kadai-core/src/main/java/io/kadai/task/api/TaskService.java

```
@@ -1631,4 +1639,18 @@ ObjectReference newObjectReference(
    * @return a {@linkplain TaskCommentQuery}
    */
   TaskCommentQuery createTaskCommentQuery();
+
+  /**
+   * Bulk updates multiple tasks by their IDs.
+   *
+   * <p>Updates only the specified fields for each task. If the user lacks permission for a task,
+   * that task is skipped and an error is reported for it, but other tasks are still updated. The
```

`skipped` may mislead.
In general: skip == discard/swallow/silent failure